Sources/TokamakCore/Views/Image.swift

```diff
@@ -17,27 +17,121 @@
 
 import Foundation
 
-public struct Image: _PrimitiveView {
-  let label: Text?
+public class AnyImageProviderBox: AnyTokenBox, Equatable {
```

Comment:
Should this have an underscore prefix to indicate that this shouldn't be re-exported?

```suggestion
public class _AnyImageProviderBox: AnyTokenBox, Equatable {
```